signac/__main__.py

```diff
@@ -185,7 +185,7 @@ def _read_index(project, fn_index=None):
     if fn_index is not None:
         _print_err("Reading index from file '{}'...".format(fn_index))
         fd = open(fn_index)
-        return (json.loads(l) for l in fd)
+        return (json.loads(i) for i in fd)
```

Comment:
This is fixed in a different PR, but swapping `l` for `i` to get the tool to be happy isn't a great reason to change it. I'm guessing it allows `i` due to the prolific use of that variable name (and most of the time it is clear what it means). Instead, use `_` to indicate that we have a temp variable that we don't care about.

Reply:
Sometimes we use `_` to just ignore the variable and here we're using that particular variable. 
I think replacing it with `line` should be fine.